docs/README.md

```diff
@@ -59,3 +65,7 @@ After building the docs with `npm run build-docs`, publishing the docs via GitHu
 1. Check the changes with `git diff` or however you prefer to view changes. One thing to look out for here is irrelevant files that got synced - you may need to clear the changes inside `web/`, adjust the list of ignored files and directories in the `.bookignore` file, and rebuild the static site if files unnecessary for the static site (which should be just HTML files, outside of the CSS, JavaScript, and fonts provided under `gitbook/`) have been copied into `web/`.
 1. Stage and commit the changes if you're happy with them.
 1. Publish to GitHub Pages by pushing your new commit(s) to the upstream `gh-pages` branch: `git push origin gh-pages`.
+
+[^a]: GitHub has since added a couple of far simpler methods, which are simply (a) including an `index.html` at the root level of your repository on `master` and choosing the appropriate repo setting to use that as the index for your GitHub Pages site or (b) including all static assets for your GitHub Pages site in a `docs/` directory on `master` (again choosing the appropriate repo setting to tell GitHub to serve the files in this directory as your static site). Eventually perhaps it will make sense for us to upgrade to the simpler `docs/` directory workflow, but at the moment with everything set up in the `gh-pages` branch workflow, investing the time to make the switch is not a high enough priority. (The `gh-pages` branch workflow also has the advantage of being somewhat cleaner when creating docs with GitBook since only the raw, source Markdown files are tracked on `master`, removing the potential for a mismatch between the source and compiled static assets, if both were tracked on `master`.)
```

Comment:
For easier navigation, I recommend adding link anchors for the footnotes.
For example, instead of `[^a]`, I suggest `<a id="footnote_a"></a>[a]`